### PR TITLE
more useful link for invite emails

### PR DESCRIPTION
**Pull request by eishay**
@gmethvin please review

### CODE REVIEW

File: shoebox/app/com/keepit/controllers/website/HomeController.scala [hunk 1]
```diff
@@ -65,8 +65,8 @@ class HomeController @Inject() (db: Database,
               from = EmailAddresses.NOTIFICATIONS,
               fromName = Some("Invitations"),
               to = EmailAddresses.INVITATION,
-              subject = s"${su.fullName} wants to be let in!",
-              htmlBody = s"Go to https://admin.kifi.com/admin/invites to accept or reject this user.",
+              subject = s"""<a href="https://admin.kifi.com/admin/user/${user.id}">${su.fullName}</a> wants to be let in!""",
```
Comment (gmethvin) [inline shoebox/app/com/keepit/controllers/website/HomeController.scala]: You shouldn't have HTML in the subject line. Other than that this is okay.

Comment (eishay) [inline shoebox/app/com/keepit/controllers/website/HomeController.scala]: fixed

**Comment by gmethvin** [comment]
lgtm

**Comment by andrewconner** [comment]
Merge conflict, @eishay

**Pull request merged**